content/docs/rosa/alb-nlb-ossm-e2e/index.md

```diff
@@ -0,0 +1,530 @@
+fi
+echo "Role ARN: $ROLE_ARN"
+# Create the policy if it doesn't exist
+EXISTING_POLICY_ARN=$(aws --region "$REGION" --output text iam list-policies --query "Policies[?PolicyName=='aws-load-balancer-operator-policy'].Arn" --output text)
+if [ -z "$EXISTING_POLICY_ARN" ]; then
+  # Create the policy if it doesn't exist
+  POLICY_ARN=$(aws --region "$REGION" --query Policy.Arn --output text iam create-policy --policy-name aws-load-balancer-operator-policy --policy-document "file://alb-operator/load-balancer-operator-policy.json")
+else
+  POLICY_ARN="$EXISTING_POLICY_ARN"
+fi
+echo "Policy ARN: $POLICY_ARN"
+aws iam attach-role-policy --role-name "${ROSA_CLUSTER_NAME}-alb-operator" \
+  --policy-arn $POLICY_ARN
+# tag cluster vpc for aws load balancer operator
+export ROSA_CLUSTER_SUBNET=$(rosa describe cluster -c $ROSA_CLUSTER_NAME -o json | jq -r '.aws.subnet_ids[0]')
+export ROSA_CLUSTER_VPC_ID=$(aws ec2 describe-subnets --subnet-ids $ROSA_CLUSTER_SUBNET --query 'Subnets[0].VpcId' --output text)
+echo "ROSA cluster VPC ID $ROSA_CLUSTER_VPC_ID"
+aws ec2 create-tags --resources ${ROSA_CLUSTER_VPC_ID} \
+  --tags Key=kubernetes.io/cluster/${CLUSTER_NAME},Value=owned \
+  --region ${REGION}
+
+# Fetch all subnets of the VPC
+export ROSA_CLUSTER_SUBNETS=$(aws ec2 describe-subnets --filters "Name=vpc-id,Values=$ROSA_CLUSTER_VPC_ID" --query 'Subnets[*].SubnetId' --output text)
+
+# Loop through each subnet and add the tag
+for subnet in $ROSA_CLUSTER_SUBNETS; do
+    aws ec2 create-tags --resources $ROSA_CLUSTER_SUBNETS --tags Key=kubernetes.io/role/internal-elb,Value=1
+done
+
+cat << EOF | oc apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: aws-load-balancer-operator
+EOF
+
+cat << EOF | oc apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+stringData:
+  credentials: |
+    [default]
+    role_arn = $ROLE_ARN
+    web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+EOF
+
+cat << EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  upgradeStrategy: Default
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  channel: stable-v1.0
+  installPlanApproval: Automatic
+  name: aws-load-balancer-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: aws-load-balancer-operator.v1.0.0
+EOF
+```
+
+ **Note:** If you have a cluster-wide proxy, you must run the following snippet or uncomment the "Configuring egress proxy for AWS Load Balancer Operator" section in the [deploy-awlbo.sh](./alb-operator/deploy-aws-lbo.sh)
```

Comment:
doesn't it also need the HTTP_PROXY, HTTPS_PROXY, NO_PROXY from the cluster wide settings ?

Reply:
No